fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/AggregatedMaterializedViewRewriter.java

```diff
@@ -126,7 +126,7 @@ protected OptExpression viewBasedRewrite(RewriteContext rewriteContext, OptExpre
         boolean isRollup = isRollupAggregate(mvGroupingKeys, queryGroupingKeys);
 
         // Cannot ROLLUP distinct
-        if (isRollup && !canRewriteForRollup(mvAggOp)) {
+        if (isRollup && mvAggOp.getAggregations().values().stream().anyMatch(callOp -> callOp.isDistinct())) {
```

Comment:
what if the mv's distinct columns is not used?

Reply:
I just revert the codes to before... The syntax is the same with before.

I think it should not be rewritten yet even for ` the mv's distinct columns is not used`?